Sources/Datadog/URLSessionAutoInstrumentation/DDURLSessionDelegate.swift

```diff
@@ -6,12 +6,25 @@
 
 import Foundation
 
+/// In case that you are not able to use `DDURLSessionDelegate` or its subclasses in your `URLSession` instance,
+/// you can use `DDURLSessionDelegateProviding` protocol.
+/// Make sure that your type implements the same `URLSessionTaskDelegate` methods as `DDURLSessionDelegate` does.
+/// Otherwise your `URLSession` will not be instrumented automatically.
```

Comment:
This is not enough to _"implement the same methods"_, the calls must be forwarded to to our delegate. How about:
```suggestion
/// In case that you are not able to use `DDURLSessionDelegate` or its subclasses in your `URLSession` instance,
/// you can use `DDURLSessionDelegateProviding` protocol.
/// Implement this protocol in your `URLSessionDelegate` class and forward required API calls to `DDURLSessionDelegate`.
```